### PR TITLE
migrate to new shib test service

### PR DESCRIPTION
**Pull request by jpeerz**
using https://samltest.id/

### CODE REVIEW

**Comment by amontenegro** [comment]
Requires this PR to be merged: https://github.com/ORCID/ORCID-Source/pull/5297

This PR will allow us to use https://samltest.id

**Comment by amontenegro** [comment]
[Enable https://samltest.id as an alternative for http://www.testshib.org/ which is disabled](https://trello.com/c/anhVs5yl/5762-enable-https-samltestid-as-an-alternative-for-http-wwwtestshiborg-which-is-disabled)

File: puppet/modules/shibboleth_nginx/files/opt/shibboleth-sp/etc/shibboleth/attribute-map.xml [hunk 1]
```diff
@@ -116,9 +116,8 @@
     <Attribute name="urn:oid:2.5.4.4" id="sn"/>
     <Attribute name="urn:oid:2.5.4.42" id="givenName"/>
     <Attribute name="urn:oid:2.16.840.1.113730.3.1.241" id="displayName"/>
-    <!--
-    <Attribute name="urn:oid:0.9.2342.19200300.100.1.1" id="uid"/>
-    <Attribute name="urn:oid:0.9.2342.19200300.100.1.3" id="mail"/>
+    <Attribute name="urn:oasis:names:tc:SAML:attribute:subject-id" id="subject-id"/>
```
Comment (wjrsimpson) [inline puppet/modules/shibboleth_nginx/files/opt/shibboleth-sp/etc/shibboleth/attribute-map.xml]: @jpeerz I think anything added here also needs to go in more_clear_input_headers (in two places) in the nginx config, for security.

Comment (amontenegro) [reply puppet/modules/shibboleth_nginx/files/opt/shibboleth-sp/etc/shibboleth/attribute-map.xml]: Ok, working on it!

Comment (amontenegro) [reply puppet/modules/shibboleth_nginx/files/opt/shibboleth-sp/etc/shibboleth/attribute-map.xml]: Fixed @wjrsimpson!

Thanks

**Comment by amontenegro** [comment]
Hey @lizkrznarich could you please merge this one?

Thanks

**Pull request merged**